source/grid/grid_in.cc

```diff
@@ -1640,7 +1640,6 @@ void GridIn<2>::read_netcdf (const std::string &filename)
 #else
   const unsigned int dim=2;
   const unsigned int spacedim=2;
-  const bool output=false;
```

Comment:
@masterleinad There is a number of uses of `output` that you apparently forgot to remove, see the build errors here:
https://cdash.kyomu.43-1.org/viewBuildError.php?buildid=1130

Could you please have a look?